### PR TITLE
Self-check script update for Chatops

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

**Comment by emedvedev** [comment]
Instead of skipping steps for RHEL7, we should just use stdout instead of grepping logs. 

If you look at the [Robot tests](https://github.com/StackStorm/st2tests/blob/master/robotfm_tests/chatops_self_check.rst), you'll see we don't parse logs anymore. That's one of the reasons I wanted to update this script in the first place. :)

File: scripts/self-check.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ Other StackStorm logs are also stored in \e[1m/var/log/st2/\e[0m.
 
 
 echo
-echo -e "Starting the Nine-Step Hubot Self-Check Program"
+echo -e "Starting the Eleven-Step Hubot Self-Check Program"
```
Comment (enykeev) [inline scripts/self-check.sh]: oh, that one is not going to scale very well...

**Comment by humblearner** [comment]
> Instead of skipping steps for RHEL7, we should just use stdout instead of grepping logs.

How about we add separate check for stdout and logs and skip logs for RHEL7 only? The reason for that is to test log generation as well. We don't want to end up debugging issues without logs. Moreover, we are mentioning in every step go and check logs. :)

**Comment by emedvedev** [comment]
Makes sense.

File: scripts/self-check.sh (hunk 1 above)
Comment (lakshmi-kannan) [inline scripts/self-check.sh]: Exactly my thoughts! lol. Just remove the "X-step"

File: scripts/self-check.sh [hunk 2]
```diff
@@ -202,17 +199,52 @@ else
 fi
 
 
-# Check that post_message is getting through.
-if [ "0" = "$(echo $hubotlogs)" ]; then
-    echo -e "\e[31mStep 9 failed: chatops.post_message hasn't been received.\e[0m"
+#Skipping these steps for RHEL7: https://github.com/StackStorm/st2-packages/issues/300
+if [ "True" = "$RH7OS" ]; then
+    echo "Step 9-10 are skipped on RHEL7";
```
Comment (lakshmi-kannan) [inline scripts/self-check.sh]: This will panic people. Just run those steps for the other OSes.

File: scripts/self-check.sh (hunk 1 above)
Comment (humblearner) [inline scripts/self-check.sh]: Will do!!

**Comment by lakshmi-kannan** [comment]
lgtm now 👍

**Comment by emedvedev** [comment]
LGTM.

**Pull request merged**